source/Octostache.Tests/CalculationFixture.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Xunit;
+
+namespace Octostache.Tests
+{
+    public class CalculationFixture : BaseFixture
+    {
+        [Theory]
+        [InlineData("3+2", "5")]
+        [InlineData("3-2", "1")]
+        [InlineData("3*2", "6")]
+        [InlineData("3/2", "1,5")]
```

Comment:
This is problematic, because of region formatting. 

[Half the world uses the decimal comma, and half use the decimal point](https://en.wikipedia.org/wiki/Decimal_separator#Usage_worldwide). Our build server happens to be configured in a region that uses a decimal point, so the build failure you saw was due to this test failing:

```
X Octostache.Tests.CalculationFixture.ConditionalIsSupported(expression: "3/2", expectedResult: "1,5")s]
Error Message:
 Expected string to be "1,5", but "1.5" differs near ".5" (index 1).
Stack Trace:
   at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
 at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
 at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
 at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
 at FluentAssertions.Primitives.StringEqualityValidator.ValidateAgainstMismatch()
 at FluentAssertions.Primitives.StringValidator.Validate()
 at FluentAssertions.Primitives.StringAssertions.Be(String expected, String because, Object[] becauseArgs)
 at Octostache.Tests.CalculationFixture.ConditionalIsSupported(String expression, String expectedResult) in 
```

Reply:
Thanks, I think I fixed it with my latest changes :).